types/parse/index.d.ts

```diff
@@ -114,6 +114,11 @@ namespace Parse {
         error?: Function;
     }
 
+    interface FieldOptions {
+      required?: boolean;
+      defaultValue?: any;
+    }
+
```

Comment:
```
interface FieldOptions<T> {
    required?: boolean;
    defaultValue?: T;
}
```

Generic type can be helpful for `defaultValue`.

Reply:
I think I'm not following you :/
How does a generic type help here?
I'm seeing the test cases in `parse-tests.ts` from [#41801](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41801/files)  but just not getting it

Reply:
Do you have an example of this use case with/without this feature maybe?

Reply:
If we use a generic type for adding `Schema` functions, it will catch an compile error when another type is used. It will prevent run time error.

```js
addBoolean(name: string, options?: Schema.SchemaOptions<boolean>)
```

Catch an error on TS
```js
schema.addBoolean('boolean', {defaultValue: 'test'}) // DefaultValue should be boolean type.
```

Reply:
Oh gotcha! I'll add it